WcaOnRails/app/models/event.rb

```diff
@@ -54,4 +54,12 @@ def fewest_moves?
   def multiple_blindfolded?
     self.id == "333mbf" || self.id == "333mbo"
   end
+
+  def serializable_hash(options = nil)
```

Comment:
In the spirit of keeping loose ends out of this PR, I've pulled out all the WCIF improvements into #1636. Let's get that merged up and then we can revisit this PR (which will be focused just on adding an API to update the WCIF).